.config/dotnet-tools.json

```diff
@@ -21,7 +21,7 @@
       ]
     },
     "microsoft.visualstudio.slngen.tool": {
-      "version": "8.1.6",
+      "version": "8.5.17",
```

Comment:
Where is this tool located? Or alternative question: Where did you get this number from?

Reply:
[n](https://www.nuget.org/packages/Microsoft.VisualStudio.SlnGen.Tool)